### PR TITLE
Getinfo

### PR DESCRIPTION
**Pull request by asfaw-michael**
@Rplesha, please review the code.

### CODE REVIEW

File: tests/test_getinfo.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                'brftab_hdr': 'lref$x1u1459il_brf.fits',
+                'tdstab': '/grp/hst/cdbs/lref/46t1623fl_tds.fits'}
+    copy_reffiles = reffiles
+    # Test
+    getinfo.resetSwitches(copy_switches, copy_reffiles)
+    # Verify
+    assert copy_switches['badtcorr'] == 'SKIPPED' # CHANGED BECAUSE ITS CORRESPONDING REF FILE IS MISSING
+    assert copy_switches['tdscorr'] == 'PERFORM' # NOT CHANGED BECAUSE ITS REF FILE EXISTS.
+    
+test_reset_switches()
```
Comment (Rplesha) [inline tests/test_getinfo.py]: remove line 275

**Review by Rplesha — COMMENTED** [review]
Looks good with the exception of removing the one line

File: tests/test_getinfo.py (hunk 1 above)
Comment (asfaw-michael) [reply tests/test_getinfo.py]: Removed

**Comment by asfaw-michael** [comment]
Fixed check failures

**Comment by Rplesha** [comment]
Some extra files got added here, too with blanks. It should only be the files that changed

**Pull request merged**